src/main/java/org/assertj/core/api/AbstractLocalDateTimeAssert.java

```diff
@@ -817,4 +818,11 @@ private static boolean haveSameYearAndMonth(LocalDateTime actual, LocalDateTime
   private static boolean haveSameYear(LocalDateTime actual, LocalDateTime other) {
     return actual.getYear() == other.getYear();
   }
+  /**
+   * Returns true if actual datetime is in a given year, false otherwise.
+   *
+   * @param actual the actual datetime. expected not be null
+   * @param year the given year. expected not be null
+   * @return true if the datetime year is equal to the given year, false otherwise
+   */
```

Comment:
at some point we will want to add `public SELF hasYear(int year)` for `LocalDateTime`, you can add it now if you feel like it or that could be done in another by you or someone else, if you don't no problem just revert the changes in this file in this case

Reply:
I will add it in another PR unless someone gets to it first.